CIP-tx-serialization-deprecation-cycles/README.md

```diff
@@ -0,0 +1,116 @@
+---
+CIP: $\pi$
+Title: Transaction Serialization Deprecation Cycle
+Authors: Jared Corduan <jared.corduan@iohk.io>
+Status: Draft
```

Comment:
Since `Status: Draft` has never really meant anything, the status for this would now be `Proposed` unless it's already active or determined in advance to be always inactive.